docs/platforms/react-native/integrations/component-names.mdx

```diff
@@ -71,6 +71,10 @@ Here's what the resulting node would look like if your bundler had applied the p
 
 The Sentry browser SDK will pick off the value from these `data` attributes and collect them when your components are interacted with.
 
+## Troubleshooting
+
+- Use `export SENTRY_LOG_LEVEL=debug` to enable debug logs. When generating the JS Bundle, you should see logs about the Sentry Metro Plugin saving and loading the default Babel transformer.
```

Comment:
```suggestion
## Troubleshooting

Use `export SENTRY_LOG_LEVEL=debug` to enable debug logs. When generating the JS Bundle, you should see logs about the Sentry Metro Plugin saving and loading the default Babel transformer.
```
Since there's only thing here for now, it doesn't really make sense to make it bulleted. Also, would it be better to call this section "Enable Debug Logs"? I'm probably missing some context, but it's not clear to me what would have to go wrong in order for "enabling debug logs" to be the solution. Maybe it would be good to add a scenario in which it would help?

Reply:
Thank you make sense, I've added more guidance. So I think now the bullet points make sense. But I'm okay leaving it as a paragraph.